### ISSUE TITLE
Find/replace doesn't seem to work with a screen reader

### ISSUE THREAD

**Issue by derekriemer** [Microsoft/vscode#28923]
Steps to Reproduce:

1. press control+h.
2. type find press tab and type replace.
3. press tab, and nothing happens. I presume there are buttons to apply the find and replace, but I can't navigate to them with a screen reader.

**Comment by zersiax** [comment]
Bump, was this ever fixed?

**Comment by alexandrudima** [comment]
I don't remember having changed anything here, but for me, pressing Tab jumps between:
* Find field
* Replace field
* Match Case toggle
* Match Whole Word toggle
* Use Regular Expressions toggle
* Previous Match action
* Next Match action
* Find in selection action
* Close
* Replace
* Replace all
* (at this point Tab leaves the find widget)

@derekriemer @zersiax Is there something needed to be done here?

**Comment by derekriemer** [comment]
I cannot reproduce this any more. Strange.
